<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>319</width>
    <height>119</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_input">
         <property name="text">
          <string>Input:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_output">
         <property name="text">
          <string>Output:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_input">
         <property name="inputMethodHints">
          <set>Qt::ImhNone</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_result">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_base">
         <property name="text">
          <string>Base:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_inputBase">
         <property name="minimum">
          <number>2</number>
         </property>
         <property name="maximum">
          <number>10</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_outputBase">
         <property name="minimum">
          <number>2</number>
         </property>
         <property name="maximum">
          <number>10</number>
         </property>
         <property name="value">
          <number>2</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
